connects/main.py

```diff
@@ -0,0 +1,115 @@
+import argparse
+import os
+import sys
+import time
+
+from databases.databases import Databases
+import alert
+
+
+def localhost_info():
+    host, dbname = '192.168.35.83', 'Sharpic'
+    user, password = 'postgres', 'secret'
+    port = 5432
+
+    return host, dbname, user, password, port
```

Comment:
https://dev.to/jakewitcher/using-env-files-for-environment-variables-in-python-applications-55a1
It would be nice to use .env files to manage environment settings such as IP addresses!